Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/app/plugin/assembler/sleigh/util/TableEntryKey.java

```diff
@@ -37,7 +38,12 @@ public class TableEntryKey implements Comparable<TableEntryKey> {
 	 */
 	public TableEntryKey(int state, AssemblySymbol sym) {
 		this.state = state;
-		this.sym = sym;
+		if (sym instanceof AssemblyWildcardTerminal) {
```

Comment:
In Java 17, you can do `sym instanceof AssemblyWildcardTerminal awt`, and skip the cast on the next line.